core/src/main/scala/org/typelevel/paiges/Chunk.scala

```diff
@@ -73,6 +73,9 @@ private[paiges] object Chunk {
       case (i, Doc.Nest(j, d)) :: z => loop(pos, ((i + j), d) :: z)
       case (_, Doc.Align(d)) :: z => loop(pos, (pos, d) :: z)
       case (i, Doc.Text(s)) :: z => ChunkStream.Item(s, pos + s.length, null, z, false)
+      case (_, l@Doc.Literal(s)) :: z =>
+        // isBreak=false because literal newlines have no indentation.
```

Comment:
I wonder why we don't want to do a find-and-replace of `\n` with `indentation + \n` here? To me, I'm a bit nervous about ignoring nesting.

It makes it weird if I want to for instance put some code in a block quote or something that expects a consistent indentation.